pkg/renderer/sgml/html5/string_test.go

```diff
@@ -51,4 +51,16 @@ var _ = Describe("strings", func() {
 </div>`
 		Expect(RenderHTML(source)).To(MatchHTML(expected))
 	})
+
+	It("title with registered unicode", func() {
```

Comment:
could you provide a test where the `unicode` attribute is not set (or set to `false`), so we can clearly see the difference in the output?

Reply:
The test just above this one has the same with it not set.

As with other boolean attributes, we don't have the notion of explicitly setting it to false.  Its enabled by its presence alone.